utils/stale-issues-and-prs.yml

```diff
@@ -0,0 +1,14 @@
+name: "Close stale Issues and PRs"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "Message to comment on stale issues. If none provided, will not mark issues stale"
```

Comment:
We should probably add some meaningful messages, instead of these. Also, it defaults to 60 days before stalling an issue/PR, and 7 more days before actually closing them. For our case, is 60 days good enough? Is it too much?

Reply:
I think it's too much!

Reply:
Fixed it! Set to 30 days + 7 days after that to close.